src/codegen/flags.rs

```diff
@@ -256,7 +256,7 @@ impl FromGlib<{sys_crate_name}::{ffi_name}> for {name} {{
     unsafe fn from_value(value: &'a {gvalue}) -> Self {{
         {assert}from_glib({glib}(value.to_glib_none().0))
     }}
-}}",
+}}unsafe impl FromOwnedValue for {name} {{}}",
```

Comment:
Please put it on its own line. Also, 
please add an empty line before it, makes it easier to spot different items.